postlammps.f90

```diff
@@ -350,14 +350,16 @@ end subroutine Block_Analysis
 
   !=================================================================================================
 
-  subroutine Print_Properties( np, property, npoints, value, interval )
+  subroutine Print_Properties( np, property, npoints, value, interval)
     integer,       intent(in) :: np, npoints
     character(sl), intent(in) :: property(np)
     real(rb),      intent(in) :: value(np,npoints)
     integer,       intent(in), optional :: interval
-    integer :: j
+    integer :: j,n=1
```

Comment:
It is not recommended to initialize n this way. Please see the Stack Overflow thread below:
http://stackoverflow.com/questions/3352741/fortran-assignment-on-declaration-and-save-attribute-gotcha